### ISSUE TITLE
Will you release the trained model？

### ISSUE THREAD

**Issue by ReeseL** [jchibane/if-net#3]
Hi:

Thanks for your contribution.
I am intreasted in human reconstruction. Will you release the trained model SVR for SingleView
Human Reconstruction？

Thanks!

**Comment by jchibane** [comment]
Hi,

you're welcome!
I uploaded you a trained model here
https://drive.google.com/file/d/1ccaYI-O4jmAOCw19bPKumTZcfYbnjBlt/view?usp=sharing

You will have to adapt the generation script such that it
1) loads your single view human data as input
2) loads the checkpoint for the SVR model

Best
Julian

**Comment by ReeseL** [comment]
Hi，
 
Thanks for your reply!
I have a question: checkpoint_epoch_6.tar could not be unzipped.
Error:
tar: This does not look like a tar archive
tar: Skipping to next header
tar: Exiting with failure status due to previous errors

If it is convenient for you, could you please upload it again?

Thanks!

**Comment by jchibane** [comment]
Hi,

you don't need to unzip this one, you can directly load it into pytorch in a similar manner to
https://github.com/jchibane/if-net/blob/master/models/generation.py#L101

I might add support of pre-trained models to the repository in a later stage, in a few weeks.

Best,
Julian

**Comment by ReeseL** [comment]
Hi,

Thanks for your answer!
 I get it.

Best,
ReeseL

**Comment by ReeseL** [comment]
Hi,

I loaded my single view human data as input and generated a full model. But, the result is not right. I am not sure the parameters I entered are correct.

I hope you can give me some advice. Thanks a lot!

parameter:
python generate.py -pointcloud -pc_samples 3000 -std_dev 0.2 0.015 -res 256 -m SVR -checkpoint 6 -batch_points 100000

input:
![input](https://user-images.githubusercontent.com/30972930/82433259-c608ff00-9ac3-11ea-81ba-bdc24fdd6b88.jpg)

result:
![results](https://user-images.githubusercontent.com/30972930/82432823-2a778e80-9ac3-11ea-82cf-b9446310d1fe.jpg)

**Comment by jchibane** [comment]
Maybe an issue with the scaling? Our human data on the hight (Y) axis goes from -0.7 to 1 approximately, so rescaling might help.

**Comment by eoozbq** [comment]
> Hi,
> 
> you're welcome! I uploaded you a trained model here https://drive.google.com/file/d/1ccaYI-O4jmAOCw19bPKumTZcfYbnjBlt/view?usp=sharing
> 
> You will have to adapt the generation script such that it
> 
> 1. loads your single view human data as input
> 2. loads the checkpoint for the SVR model
> 
> Best Julian

hello,,i'm sorry,  can I ask a link now?
